spec/models/user_spec.rb

```diff
@@ -22,4 +22,11 @@
     end
   end
 
+  describe '#generate_token' do
+    it 'generates a random token when user is created' do
```

Comment:
This should be rather - it generates token when you call `.generate_token` after user clicks a password reset.